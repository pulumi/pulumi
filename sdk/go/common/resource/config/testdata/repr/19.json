{"my:testKey":[{"inner":{"secure":"securevalue"}},{"secure":"securevalue2"}]}