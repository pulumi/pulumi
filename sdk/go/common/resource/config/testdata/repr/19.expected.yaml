decrypt:
  my:testKey: '[{"inner":{"secure":"securevalue"}},{"secure":"securevalue2"}]'
paths:
  my:testKey:
    value:
    - inner:
        secure: securevalue
    - secure: securevalue2
    string: '[{"inner":{"secure":"securevalue"}},{"secure":"securevalue2"}]'
    redacted: '[{"inner":"[secret]"},"[secret]"]'
    object: HRAADltdaW50ZXJmYWNlIHt9/4MCAQL/hAABEAAA/63/hCgAAhdtYXBbc3RyaW5nXWludGVyZmFjZSB7ff+BBAEC/4IAAQwBEAAA/4D/gkIAAQVpbm5lchdtYXBbc3RyaW5nXWludGVyZmFjZSB7ff+CHwABBnNlY3VyZQZzdHJpbmcMDQALc2VjdXJldmFsdWUXbWFwW3N0cmluZ11pbnRlcmZhY2Uge33/giAAAQZzZWN1cmUGc3RyaW5nDA4ADHNlY3VyZXZhbHVlMg==
    secure: true
    isObject: true
    secureValues:
    - securevalue
    - securevalue2
  my:testKey[0]:
    value:
      inner:
        secure: securevalue
    string: '{"inner":{"secure":"securevalue"}}'
    redacted: '{"inner":"[secret]"}'
    object: KBAAF21hcFtzdHJpbmddaW50ZXJmYWNlIHt9/4EEAQL/ggABDAEQAABF/4JCAAEFaW5uZXIXbWFwW3N0cmluZ11pbnRlcmZhY2Uge33/gh8AAQZzZWN1cmUGc3RyaW5nDA0AC3NlY3VyZXZhbHVl
    secure: true
    isObject: true
    secureValues:
    - securevalue
  my:testKey[0].inner:
    value:
      secure: securevalue
    string: securevalue
    redacted: '[secret]'
    object: GBAABnN0cmluZwwNAAtzZWN1cmV2YWx1ZQ==
    secure: true
    isObject: false
    secureValues:
    - securevalue
  my:testKey[1]:
    value:
      secure: securevalue2
    string: securevalue2
    redacted: '[secret]'
    object: GRAABnN0cmluZwwOAAxzZWN1cmV2YWx1ZTI=
    secure: true
    isObject: false
    secureValues:
    - securevalue2
